sunpy/timeseries/sources/goes.py

```diff
@@ -56,6 +57,10 @@ class XRSTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'xrs'
 
+    _netcdf_read_kw = {}
+    if h5netcdf.__version__ >= LooseVersion("0.9"):
+        _netcdf_read_kw['decode_strings'] = True
```

Comment:
Not sure if this is the best place to put these lines, other suggestions welcome.

Reply:
Why not just no this in the `_parse_netcdf` method?

Reply:
That's what I originally did, but it's needed in `is_datasource_for` too.